src/tests/query/error.rs

```diff
@@ -271,3 +271,65 @@ async fn garbage_datetime_values(api: &mut dyn TestApi) -> crate::Result<()> {
 
     Ok(())
 }
+
+#[test_each_connector]
+async fn should_pick_up_partially_failed_raw_cmd_scripts(api: &mut dyn TestApi) -> crate::Result<()> {
+    let conn = api.conn();
+
+    let result = conn.raw_cmd("SELECT YOLO; SELECT 1;").await;
+
+    assert!(result.is_err());
+
+    let result = conn.raw_cmd("SELECT 1; SELECT NULL; SELECT YOLO; SELECT 2;").await;
+
+    assert!(result.is_err());
+
+    if api.conn().connection_info().sql_family().is_mysql() {
+        let error_message = result.unwrap_err().to_string();
+        assert_eq!(error_message, "Error accessing result set, column not found: YOLO");
+    }
+
+    Ok(())
+}
+
+#[test_each_connector(tags("mysql"))]
+async fn should_execute_multi_statement_queries_with_raw_cmd(api: &mut dyn TestApi) -> crate::Result<()> {
```

Comment:
This test is challenging to make cross-database, the create table abstraction in `TestApi` doesn't really work because we specifically want to test multiple statements in the same script. So it would have a bunch of database-specific code, or a new create table abstraction.

Reply:
Something similar to this? https://github.com/prisma/quaint/blob/master/src/tests/test_api/mysql.rs#L43-L56

Reply:
It wouldn't test what we want to test, because we want the 4 statements in one call.

Reply:
We could have an additional function to that...